src/educational-technology.bib

```diff
@@ -0,0 +1,86 @@
+@article{guzdial2004programming,
+  title={Programming environments for novices},
+  author={Guzdial, Mark},
+  journal={Computer science education research},
+  volume={2004},
+  pages={127--154},
+  year={2004}
+}
+
+@article{kelleher2005lowering,
+  title={Lowering the barriers to programming: A taxonomy of programming environments and languages for novice programmers},
+  author={Kelleher, Caitlin and Pausch, Randy},
+  journal={ACM Computing Surveys (CSUR)},
+  volume={37},
+  number={2},
+  pages={83--137},
+  year={2005},
+  publisher={ACM}
+}
+
+@article{wyeth2001electronic,
+  title={Electronic blocks: Tangible programming elements for preschoolers},
+  journal={},
```

Comment:
what is the venue?

Reply:
I wasn't able to find a venue it was published in

Reply:
Proceedings of the Eighth IFIP TC13 Conference on Human-Computer Interaction (https://www.researchgate.net/publication/43462811_Electronic_blocks_Tangible_programming_elements_for_preschoolers/citation/download)